compiler/p/codegen/InstOpCode.hpp

```diff
@@ -25,7 +25,7 @@
 #include "codegen/OMRInstOpCode.hpp"
 
 #ifndef OPS_MAX
-#define OPS_MAX    TR::InstOpCode::PPCNumOpCodes
+#define OPS_MAX    TR::InstOpCode::NumOpCodes
```

Comment:
Sorry, can't find where NumOpCodes is defined and if it has different value from old PPCNumOpCodes.

Reply:
I think it would be helpful to clarify the goal of this particular PR, otherwise it takes a bit of time to figure out:

1) not to add any new opcodes, the number of opcodes stays the same for now and enums don't change
2) generate new properties table for the same enums
3) add verification code that compares new and old tables
4) new properties table is not used yet

Is that correct?

Reply:
That's correct. I'll amend the PR with additional info as per the request. I'll be tacking this tomorrow.

Reply:
> Sorry, can't find where NumOpCodes is defined and if it has different value from old PPCNumOpCodes.

`NumOpCodes` is defined in [1] in the common code generator.

[1] https://github.com/eclipse/omr/blob/e2478dca0870e9410ea9c93b4c0b8a5bd057fda9/compiler/codegen/OMRInstOpCode.hpp#L46-L50

Reply:
Added some more documentation to the PR. All bullet points are now complete and we have switched to using the consolidated tables with the concatenation of all the commits within this PR.